Formula/ansible.rb

```diff
@@ -59,40 +60,24 @@ class Ansible < Formula
   end
   ### end
 
-  resource "Babel" do
```

Comment:
Because of the beast `ansible` is I left the ones explicitly labelled as needing to live up top where they were, and then slowly glued together the various resource lists from the 13/14 components in alphabetical order myself, so `poet` spat out the lists here and I organised them. Blame lies with me 😸.